base/reflection.jl

```diff
@@ -199,14 +199,14 @@ function _dump_function(f, t::ANY, native, wrapper, strip_ir_metadata)
         str = ccall(:jl_dump_function_asm, Any, (Ptr{Void},), llvmf)::ByteString
     else
         str = ccall(:jl_dump_function_ir, Any,
-                    (Ptr{Void}, Bool), llvmf, strip_ir_metadata)::ByteString
+                    (Ptr{Void}, Bool, Bool), llvmf, strip_ir_metadata, dump_module)::ByteString
     end
 
     return str
 end
 
-code_llvm(io::IO, f::Function, types::ANY, strip_ir_metadata=true) =
-    print(io, _dump_function(f, types, false, false, strip_ir_metadata))
+code_llvm(io::IO, f::Function, types::ANY, strip_ir_metadata=true, dump_module=false) =
```

Comment:
Yes, it would have. I forgot that this was still pending and somebody told be their build was broken because of this so I wanted to get it in. Will fix. I guess the name remains to be decided. I'm not a fan of `full_module` or `complete_module` because the implies that it shows some sort of complete module, when really it shows a barebones module that we copy the function into. `with_module` perhaps?